4/debian-10/rootfs/opt/bitnami/scripts/libpgpool.sh

```diff
@@ -498,6 +503,11 @@ pgpool_create_config() {
         info "Custom configuration '$PGPOOL_USER_CONF_FILE' detected!. Adding it to the configuration file."
         cat "$PGPOOL_USER_CONF_FILE" >>"$PGPOOL_CONF_FILE"
     fi
+
+    if [[ -f "$PGPOOL_USER_HBA_FILE" ]]; then
+        info "Custom configuration '$PGPOOL_USER_HBA_FILE' detected!. Overwriting the generated hba file."
+        cat "$PGPOOL_USER_HBA_FILE" >"$PGPOOL_PGHBA_FILE"
```

Comment:
In order to follow the same logic as the pgpool conf file, why not use the same format?
```suggestion
        info "Custom configuration '$PGPOOL_USER_HBA_FILE' detected!. Adding it to the configuration file."
        cat "$PGPOOL_USER_HBA_FILE" >>"$PGPOOL_PGHBA_FILE"
```

Reply:
Maybe I'm wrong, but I think that if we only append the custom hba to the default hba, the default hba entries are always considered first and the following custom hba entries are ignored.

> There is no "fall-through" or "backup": if one record is chosen and the authentication fails, subsequent records are not considered. If no record matches, access is denied.

https://www.pgpool.net/docs/latest/en/html/auth-pool-hba-conf.html

For example:

If the default hba allows md5 and the custom hba defines trust, it wouldn't work:

```
host    all         all         0.0.0.0/0             md5
host    all         all         0.0.0.0/0             trust
```

Reply:
Yes, you're right, I didn't know how it works behind scenes